onnx/defs/nn/defs.cc

```diff
@@ -1416,4 +1402,68 @@ ONNX_OPERATOR_SET_SCHEMA(
         .SetDoc(LRN_ver1_doc)
         .TypeAndShapeInferenceFunction(propagateShapeAndTypeFromFirstInput));
 
+static const char* StringNormalizer_ver9_doc = R"DOC(
+[optional] Step1: Remove elements in X if they match any of the stop words so
+ that the output tensor will not contain any stop words. This operator only accepts [C]-
+ and [1, C]-tensors. If all elements in X are dropped, the output will be the default value
+ of string tensor with shape [1] if input shape is [C] and shape [1, 1] if input shape is [1, C].
+[optional] Step2: Lower all characters (if action is LOWER) in X or capitalize them (when action is UPPER).
+)DOC";
+
+ONNX_OPERATOR_SET_SCHEMA(
+    StringNormalizer,
+    9,
+    OpSchema()
+        .Input(0, "X", "Strings to normalize", "T")
+        .Output(0, "Y", "Normalized strings", "T")
+        .TypeConstraint(
+            "T",
+            {"tensor(string)"},
+            "Input/Output is a UTF-8 string tensor")
+        .Attr(
+            "casechangeaction",
```

Comment:
do you want to make them as "optional" attribute with default value? say, casechangeaction with default value "NONE"; and is_case_sensitive with default value "false".

Reply:
Sound reasonable to me. :)

Reply:
attribute named as "case_change_action" (consistent with the other attribute)? btw, how do you think of my comments of making them as an optional attribute with default value?

Reply:
I will make this change